src/Repositories/MigrationRepository.php

```diff
@@ -48,9 +48,17 @@ public function delete(string $fileName): bool
             ->delete();
     }
 
+    public function deleteAll(): void
+    {
+        $this->table()->delete();
+    }
+
+    /**
+     * @deprecated
+     */
     public function truncate(): void
     {
-        $this->table()->truncate();
+        $this->deleteAll();
```

Comment:
I think we should keep the original code here. We've deprecated `truncate` instead of deleting it completely because it is a public method and therefore it might be used by someone.

Reply:
Ah right! Okay I’ll update the PR 😄